docs/tutorials/add-prisma-database.md

```diff
@@ -0,0 +1,359 @@
+
+# External Prisma database support for a Seven add-on
+
+This tutorial will show you how to add a small "Like" counter button to any Seven content object.
+It will persist its value in an external database using {term}`Prisma`.
```

Comment:
This needs a definition in `plone/documentation`'s glossary before it can be merged. See https://app.readthedocs.org/projects/volto/builds/29831361/#288836463--290.

What is Prisma? I honestly tried to find a definition on Prisma's site, and I fell into a deep rabbit hole of Prisma this-and-that.